solutionbox/inception/release.sh

```diff
@@ -14,6 +14,8 @@
 
 # Build a distribution package
 python setup.py sdist --formats=gztar
+
+# Release to gs
 gsutil cp ./dist/inception-0.1.tar.gz gs://cloud-datalab/packages/inception-0.1.tar.gz
```

Comment:
why is this file in github? I mean, not everyone can upload a file to that location, right?